examples/connector/python/build.sh

```diff
@@ -6,13 +6,18 @@ source connector_run/bin/activate
 pip install -r requirements.txt
 
 # copying protos present in the root of directory to `protos` folder
-mkdir -p protos
-cp ../../../*.proto protos/
+mkdir -p proto
+for file in ../../../*v2.proto; do
+    file_name=$(basename "$file")
+    new_file="proto/${file_name%_v2.proto}.proto"
```

Comment:
why have we changed the copy target from `protos/` to `proto/`?

Reply:
good catch

Reply:
Just to be consistent with other examples. We named it as `proto` in there.

Reply:
All the examples should be using these new `xxx_v2.proto` files